examples/make_eeg_bids.py

```diff
@@ -0,0 +1,181 @@
+"""
+=========================
+Use MNE-Bids for EEG Data
+=========================
+
+In this example, we use MNE-Bids to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+1. Download some EEG data from the Open Science Framework (https://osf.io)
+2. Load the data, extract information, and save in a new BIDS directory
+3. Check the result and compare it with the standard
+
+"""
+
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+# License: BSD (3-clause)
+
+# We are importing everything we need for this example:
+import os
+
+from mne import find_events
+from mne.io import read_raw_brainvision
+from mne.utils import _TempDir
+
+from mne_bids import raw_to_bids
+from mne_bids.datasets import download_matchingpennies_subj
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the example data that is
+# provided with the report on the BIDS extension for EEF data: The "Matching
+# Pennies" dataset. See here for more information: https://osf.io/cj2dr/
+#
+# For convenience, MNE-Bids has a function to download this data.
+
+# make a temporary directory to save the data to
+data_dir = _TempDir()
```

Comment:
don't expose private function to user

Reply:
Mmmh why not? The alternative would be to do some lines of codes with `os.mkdir`, potentially with an awkward `if os.path.exists`, or else making some hash to be sure it doesn't exist using `np.random.bytes()` --> seems longer and achieves the same

Reply:
private function should be private :-) you can save it in ~\mne_data\ if you like